src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -2783,6 +2788,17 @@ bool Creature::LoadCreaturesAddon(bool reload)
     return true;
 }
 
+void Creature::LoadSparringPct()
```

Comment:
On second thought I don't get why we need this. Why can't we just get the sparringpct from SparringData directly in GetSparringPct

Reply:
If we do this, you'll be performing an operation every time GetSparringPct is called (and with DealDamage it's call very often). I think it's better to load a sparringPct variable for all creatures and check its state as many others variables (spawnId, guid...).

Reply:
Load it into store at server startup